__tests__/github-markdown_test.js

```diff
@@ -6,29 +6,29 @@ const results = {
       beforeStats: 8914,
       afterStats: 3361,
       percentChange: -62.29526587390622,
-      compressionWasSignificant: true
+      compressionWasSignificant: true,
```

Comment:
Should we add `prettier` to avoid these formatting changes popping up in PRs in the future?

Reply:
Yep. I didn’t want to do a mega formatting change on this PR but that’s exactly what I’d like to see happen. Same config as elsewhere 👍